src/components/atoms/Modal/Modal.tsx

```diff
@@ -0,0 +1,58 @@
+import { FC, ReactNode, PropsWithChildren } from 'react'
```

Comment:
**[blocker]**

Currnetly it seems that we only have 1 type of modal semi-implemented, which is the narrow modal, with wrong bottom spacing + with the wrong close button (The close button that should be used on another type of modal).

This components should be much easier to use, currently whenever we want to use it we need to pass in the full footer component, the full header component + a bunch of extra styles to create the modal we want. This then has to be done every time we want to use a modal. 

I would prefer if more of that work would be done inside this component. 

We essentially have:

1. 3 different sizes for modal (narrow, medium and big), currently the component only supports the narrow version out of the box. Would be good if this could just be picked with 1 prop, instead of passing in a bunch of styles, a new footer and new header.

2. 3 different types of footers (buttons split, buttons on the right, no footer). Currently we also need to manually create the component every time and pass it in, which will get quite troublesome, since we have a lot of Modals in this project. Would be good if this could also be switched with a prop. Possibly with extra props for button labels + what they will do.

3. Close button either visible or not (this seems to be handled)


1 option would be to rewrite this component to match and another option would be to basically use this as a modal basis and creating a new Modal component ,which we will actually use, which itself uses the modal bases and maps out all the the different types of modals we will have.

Reply:
Yes, made the modal more easier to use. Ready for review